client/controller/index.web.js

```diff
@@ -46,6 +49,17 @@ export const ProviderWrappedLayout = ( {
 } ) => {
 	const state = store.getState();
 	const userLoggedIn = isUserLoggedIn( state );
+	const [ isLoading, experimentAssignment ] = useExperiment(
```

Comment:
It seems like we're handling experiment assignment here to have the information prepared for the features-list as soon as possible. Does that sound right?

I ask because, if I recall correctly, we used to typically place this kind of code in `plans-features-main`. Is there any downside to following the previously established conventions?

Either way, this implementation seems reasonable to me. It's also worth noting for posterity that this experiment code is temporary because of the nature of experiments.

Reply:
It's fascinating to see that you think holistically regarding the affecting scope of the feature list, @ddc22 :) 

Ideally it will be the best that whoever sees the new feature copies will always see the same set of copies. However, this experiment is meant to be restricted within within `/start/plans` and the logged-in `/plans`, so it's expected that there will be inconsistent copies for the treatment group. Also, if we put `useExperiment` here, it will be very hard to limit the assignment to the `/start` only traffic, since there is no signup flow concept at this point. It's also concerning to introduce one extra async call for every dotcom user. If we ever want to do a global-scope experiment like this, we should do that from the backend.

Thus, let's revert it back within `plan-features-main` as it is now.